fleet-management-service/src/main/java/cz/fi/muni/pa165/service/JourneyServiceImpl.java

```diff
@@ -44,4 +59,19 @@
         return journeysDate;
     }
 
+    public void createJourney(Long vehicleId, Long employeeId, Date startDate) {
+        Employee employee = employeeDao.findById(employeeId);
+        Vehicle vehicle = vehicleDao.findById(vehicleId);
+        Journey jouney = new Journey(startDate, vehicle, employee);
+        journeyDao.persist(jouney);
```

Comment:
I thought that service layer should work wiht entities and facade with DTOs. I tried to avoid addtional mappings.